pkgs/applications/audio/patchance/default.nix

```diff
@@ -29,6 +30,9 @@ buildPythonApplication rec {
 
   postFixup = ''
     wrapPythonProgramsIn "$out/share/patchance/src" "$out $pythonPath"
+    for file in $out/bin/*; do
+      wrapQtApp "$file"
```

Comment:
do
```nix
preFixup = ''
  makeWrapperArgs+=("''${qtWrapperArgs[@]}")
'';
```
to not double-wrap

Reply:
@Artturin would this be good to apply to #236762?

Reply:
Yes

Also this for file can now be removed

Reply:
When I remove the loop, I get the "could not find xcb" error.

Reply:
@Artturin ping

Reply:
Ok it's the manual running is still needed because the auto qt wrap checks for binaries